src/main/scala/mesosphere/marathon/api/v2/json/Formats.scala

```diff
@@ -138,6 +139,11 @@ trait Formats
     Writes[PathId] { id => JsString(id.toString) }
   )
 
+  implicit lazy val TaskIdFormat: Format[Task.Id] = Format(
+    Reads.of[String](Reads.minLength[String](3)).map(Task.Id(_)),
```

Comment:
Admittedly, a fairly random choice. Basically, we did not validate task IDs before. Do we also want such a high length limit during testing? If the ID doesn't match an existing task, it is mood anyways.

 I think we do not really need to verify it here but my reasoning was that there needed to be at least an app prefix, a dot, and a uuid.

I can use `36` if it doesn't break any tests.